gen/definitions/dynamic_object.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: DynamicObject
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/dynamicobjects
+data_source_name_query: true
+doc_category: Objects
+attributes:
+  - model_name: name
+    type: String
+    mandatory: true
+    description: User chosen resource name.
+    example: DynamicObject1
```

Comment:
If I create the resource and then try to modify the `name`, I get
```
StatusCode 400,
│ {"error":{"category":"FRAMEWORK","messages":[{"description":"Cannot modify dynamic
│ object name"}],"severity":"ERROR"}}
```
Please always verify modifications manually. This attribute needs a `requires_replace: true` probably.